crates/net/network-types/src/peers/mod.rs

```diff
@@ -136,3 +140,30 @@ impl Peer {
         matches!(self.kind, PeerKind::Static)
     }
 }
+
+/// Information about the range of blocks available from a peer.
+#[derive(Debug, Clone)]
+pub struct RangeInfo {
+    /// The inner range information.
+    inner: Arc<RangeInfoInner>,
+}
```

Comment:
this we should move the the reth-network crate because this is reth-network internal type and this crate should remain no-std